language-configuration.json

```diff
@@ -1,8 +1,8 @@
 {
   "comments": {
-    // symbol used for single line comment. Remove this entry if your language does not support line comments
-    "lineComment": ["#", "//"],
```

Comment:
My linter is complaining that this can't be an array (because of [this interface](https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/languages/languageConfiguration.ts#L17), so I chose one. I don't think `["#", "//"]` has any effect.